### ISSUE TITLE
Visual Studio crashes when deploying to Windows Phone

### ISSUE THREAD

**Issue by mcosmin222** [dotnet/roslyn#12181]
**Version Used**: 

2015 Update 2, Update 3 RC

**Steps to Reproduce**:

1. Create basic UWP app
2. Try deploying to windows phone device 10
3. in like 90% of tries, VS crashes; if you manage to deploy once, it will continue to work after that

**Expected Behavior**:
To deploy the app without any problem.

**Actual Behavior**:
Crashes in 90% of tries. Once you can deploy it, it will continue to work for subsequent deploys, until you restart VS. 

This has been going on for 6 months now, and Microsoft doesn't seem to care.

**Comment by Pilchie** [comment]
Hi,

Thanks for the report - unfortunately the Roslyn project has nothing to do with the deployment of UWP apps.  I suggest filing this report on http://connect.microsoft.com.

**Comment by alexanderdp** [comment]
I never had that issue with Windows 10 UWP, but I did with Windows Phone 8, this is what I did to fix it, maybe you can try the same or similar for your UWP SDKs

This solved my Windows Phone 8.1 Deploy to Device Visual Studio 2015 crash issue:

1) Shut down Visual Studio.


2) After enabling/unlocking your device (WinPhone 8.)1 for development;
msdn.microsoft.com/en-us/library/windows/apps/ff769508

Make sure you download and install the Windows Phone 8.1 SDK.
developer.microsoft.com/en-us/windows/downloads/windows-8-1-sdk


3) After installing this SDK, try to connect to your WinPhone 8.1 through the SDK, which will trigger a "Special Update" for that device.

4) After installing the update on your device and it reboots, start Visual Studio and deploy/debug to your device.

**Comment by bokova** [comment]
This might help, then again might not:
I tried to deploy to a windows phone 8 (lumia denim) using VS Community 2015. VS crashed each time I tried to deploy. Looks like the phone needed a special update. VS didn't prompt me to update, neither did the phone. What worked was running the "Windows phone developer power tools" utility that came along with VS/windows phone 8.1 SDK. There I tried to connect the phone. That triggered the "Phone Tools Update Pack" that started th installation on the phone.
